src/Compilers/VisualBasic/Portable/Syntax/VisualBasicSyntaxTree.ParsedSyntaxTree.vb

```diff
@@ -123,13 +123,6 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 Return New SimpleSyntaxReference(Me, node)
             End Function
 
-            ''' <summary>
-            ''' Returns a <see cref="String" /> that represents the source code of this parsed tree.
-            ''' </summary>
-            Public Overrides Function ToString() As String
-                Return Me.GetText().ToString()
-            End Function
-
```

Comment:
Is this actually a binary breaking change.  I know C# emits calls to the original virtual, but doesn't VB emit calls to the most derived override known statically?

(Which implies a bug in our public changes analyzer if I'm right).